model/item.cc

```diff
@@ -213,11 +213,15 @@ namespace minsky
   
   void Item::drawResizeHandles(cairo_t* cairo) const
   {
-    double sf=portRadiusMult*zoomFactor();  
-    drawResizeHandle(cairo,right()-x(),top()-y(),sf,0.5*M_PI);
-    drawResizeHandle(cairo,left()-x(),top()-y(),sf,M_PI);
-    drawResizeHandle(cairo,left()-x(),bottom()-y(),sf,1.5*M_PI);
-    drawResizeHandle(cairo,right()-x(),bottom()-y(),sf,0);
+    double sf=portRadiusMult*zoomFactor();
+    // Ops, vars and switch icon only resize from bottom right corner. for ticket 1203
+    if (!switchIconCast() && !variableCast() && (!operationCast() || operationCast()->type()==OperationType::ravel))  
```

Comment:
This is a case where we can refactor in terms of virtual functions, but let's leave it as is for now, and add another ticket to refactor this in terms of virtual functions.